src/explore-education-statistics-common/src/services/__tests__/tableBuilderService.test.ts

```diff
@@ -0,0 +1,479 @@
+import tableBuilderService, {
+  SubjectMeta,
+  TableDataResponse,
+  TableDataResult,
+} from '@common/services/tableBuilderService';
+import { dataApi as _dataApi } from '@common/services/api';
+import * as _combineMeasures from '@common/services/util/combineMeasuresWithDuplicateLocationCodes';
+
+jest.mock('@common/services/api');
+jest.mock('@common/services/util/combineMeasuresWithDuplicateLocationCodes');
+
+const dataApi = _dataApi as jest.Mocked<typeof _dataApi>;
+const combineMeasures = _combineMeasures as jest.Mocked<
+  typeof _combineMeasures
+>;
+
+describe('tableBuilderService', () => {
+  test('locations with duplicate levels and codes are merged in getSubjectMeta()', async () => {
+    const originalMeta: SubjectMeta = {
```

Comment:
Could move originalMeta up out of this test block so don't have to duplicate it in the next test.

Reply:
Yup!

Reply:
Did it for all the tests and made it much more succinct, thanks!